test/graphql/attach_files_to_sample_test.rb

```diff
@@ -47,13 +59,39 @@ def setup
     assert_equal 'afts.fastq', sample.attachments[0].filename.to_s
   end
 
+  test 'attachFilesToSample mutation should work with valid params, puid, and api scope token' do
+    sample = samples(:sampleJeff)
+    blob_file = active_storage_blobs(:attachment_attach_files_to_sample_test_blob)
+
+    assert_equal 0, sample.attachments.count
+
+    result = IridaSchema.execute(ATTACH_FILES_TO_SAMPLE_BY_SAMPLE_PUID_MUTATION,
+                                 context: { current_user: @user, token: @api_scope_token },
+                                 variables: { files: [blob_file.signed_id],
+                                              samplePuid: sample.id })
```

Comment:
This should be `sample.puid`

Reply:
Fixed in https://github.com/phac-nml/irida-next/commit/c362e774144acf1ee4ec137f878f09c4b12a154f good catch!